### DISCUSSION TITLE
Inconsistent R interpreter discovery

### DISCUSSION THREAD

**Discussion by infoinfo64** [posit-dev/positron#4316] [Q&A] [answered]
I'm really loving Positron, but I've recently observed some strangeness with R interpreter discovery. I have two versions of R installed (c:/windows/program files/R/R-4.3.3, and c:/windows/program files/R/R-4.4.1). Some projects can see only 4.3.3, some can only see 4.4.1, and new projects can see neither version.  This behavior only appeared under Positron version 2024.08.0-05, and i see the same behavior in 2024.08.0-24. Anyone else seeing this? I've tried uninstalling and reinstalling 4.4.1 and Positron, but this behavior persists. I'll add that having the option to manually browse for and select an interpreter would be helpful.

**Comment by infoinfo64** [comment]
Sorry, just seeing this resolution in another thread - https://github.com/posit-dev/positron/issues/4280#issuecomment-2279026209

**Comment by jennybc** [comment, marked as answer]
This is fixed in today’s release.
